src/model/HeatExchangerAirToAirSensibleAndLatent.cpp

```diff
@@ -744,5 +804,196 @@ namespace model {
     return getImpl<detail::HeatExchangerAirToAirSensibleAndLatent_Impl>()->autosizedNominalSupplyAirFlowRate();
   }
 
+  // DEPRECATED
+  double HeatExchangerAirToAirSensibleAndLatent::sensibleEffectivenessat75HeatingAirFlow() const {
+    DEPRECATED_AT_MSG(3, 8, 0, "As of EnergyPlus 24.1.0, this property ...");
+    if (boost::optional<TableLookup> sensibleEffectivenessofHeatingAirFlowCurve_ =
+          sensibleEffectivenessofHeatingAirFlowCurve()->optionalCast<TableLookup>()) {
+      return sensibleEffectivenessofHeatingAirFlowCurve_->outputValues()[0];
+    } else {
+      return getImpl<detail::HeatExchangerAirToAirSensibleAndLatent_Impl>()->sensibleEffectivenessat100HeatingAirFlow();
+    }
+  }
```

Comment:
* You have no proof that the `outputValues()[0] `corresponds to X=0.75. If I created that TableLookup manually, this may very well **NOT** be the case.

* You MUST check if the `sensibleEffectivenessofHeatingAirFlowCurve()` is initialized before you deference it or it'll throw if not. Your testing didn't catch it because the curves are optional BUT your constructor is initializing them.

* If I have set something other than a TableLookup (meaning a regular Curve:XXX object), you're also returning the 100 airflow without warning.

Reply:
All the getters would work fine if we had `TableLookup_Impl::evaluate`, but we don't.